environments/dev.yml

```diff
@@ -22,3 +22,7 @@ dependencies:
   - flake8
   - pip
   - pytest
+  - pip:
+    - mkdocs
```

Comment:
seems like mkdocs and material are on conda forge, I think it is better than to include them directly from there then

Reply:
Are you somehow using mkdocs and pymdown-extensions or are they just dependencies?
Are they not just automatically installed? I would only add mkdocs-material if the others are just dependencies

Reply:
I switched to install them via conda-forge. Good point, I am not sure about dependency relationship.